srcpkgs/libshumate/template

```diff
@@ -1,21 +1,24 @@
 # Template file for 'libshumate'
 pkgname=libshumate
-version=1.1.3
+version=1.2.0
 revision=1
 build_style=meson
 build_helper="gir"
-configure_args="$(vopt_bool gir vapi) $(vopt_bool gir gir) $(vopt_bool gir gtk_doc)"
-hostmakedepends="pkg-config $(vopt_if gir 'gi-docgen vala') glib-devel gettext"
-makedepends="libglib-devel sqlite-devel cairo-devel gtk4-devel libsoup3-devel"
+configure_args="$(vopt_bool gir vapi) $(vopt_bool gir gir)
+ $(vopt_bool gir gtk_doc) --buildtype debugoptimized"
```

Comment:
Don't use `--buildtype debugoptimized` we use `--buildtype plain` because we don't want extra junk to pollute the cflags.

Just use vsed to remove `-DG_DISABLE_ASSERT`.

Reply:
thanks for a better fix.

add to the template

```sh
post_patch() {
	vsed -i shumate/meson.build -e '/DG_DISABLE_ASSERT/d'
}
```

 and remove `--buildtype debugoptimized`

Reply:
honestly, i should've found that option myself
looked in the wrong subdir meson.build (tests)